src/main/java/service/FruitStoreService.java

```diff
@@ -0,0 +1,38 @@
+package service;
+
+import db.FruitStorage;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import strategy.BalanceStrategy;
+import strategy.PurchaseStrategy;
+import strategy.ReturnStrategy;
+import strategy.SupplyStrategy;
+import strategy.TransactionStrategy;
+
+public class FruitStoreService {
+    private Map<FruitTransaction.Operation, TransactionStrategy> strategyMap;
+
+    public FruitStoreService() {
+        strategyMap = Map.of(FruitTransaction.Operation.BALANCE, new BalanceStrategy(),
+                FruitTransaction.Operation.SUPPLY, new SupplyStrategy(),
+                FruitTransaction.Operation.PURCHASE, new PurchaseStrategy(),
+                FruitTransaction.Operation.RETURN, new ReturnStrategy());
+    }
+
+    public FruitStorage processTransactions(List<FruitTransaction> transactions) {
+        FruitStorage fruitStorage = new FruitStorage();
```

Comment:
In order to have flexible solution you should obtain things you rely on in constructor. So your service can be created to work with any map and any storage that may be created in future:
```suggestion
    public FruitStoreService(Map<FruitTransaction.Operation, TransactionStrategy> strategyMap, FruitStorage fruitStorage ) {
        this.strategyMap = strategyMap;
        this.fruitStorage = fruitStorage;
    }

    public FruitStorage processTransactions(List<FruitTransaction> transactions) {
```
map and fruitStorage should be initialized in main class and passed as arguments